features/0510-dif-pres-exch-attach/README.md

```diff
@@ -241,7 +314,158 @@ Complete message example:
+            "number": "34DGE352",
+            "dob": "07/13/80"
+          }
+        },
+        "proof": {
+            "type": "RsaSignature2018",
+            "created": "2017-06-18T21:19:10Z",
+            "proofPurpose": "assertionMethod",
+            "verificationMethod": "https://example.edu/issuers/keys/1",
+            "jws": "..."
+        }
+    }],
+    "proof": {
+      "type": "Ed25519Signature2018",
+      "proofPurpose": "assertionMethod",
+      "created": "2017-09-23T20:21:34Z",
+      "verificationMethod": "did:example:123456#key1",
+      "challenge": "2bbgh3dgjg2302d-d2b3gi423d42",
+      "domain": "example.org",
+      "jws": "eyJ0eXAiOiJK...gFWFOEjXk"
+  }
+}
+```
+</details>
+
+#### BbsBlsSignatureProof2020
 
+**Input Parameters**:
+* `presentation_definition.input_descriptors.constraints.limit_disclosure`: `required`
+* `presentation_definition.format`: `ldp_vc`
+* `presentation_Definition.format.proof_type`: `BbsBlsSignatureProof2020`
+* `options.nonce`: a base64-encoded random sequence of bytes
+
+**Result:**
+
+A [Verifiable Presentation](https://www.w3.org/TR/vc-data-model/#presentations-0) of type
+[Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) containing the
+credentials requested under the `verifiableCredential` property. No `proof` is required on the Verifiable Presentation
+itself since the enclosed credentials with proof type `BbsBlsSignatureProof2020` will contain the proofs.
+
+<details><summary>Example</summary>
+
+```json
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/v2",
+        "https://w3id.org/security/bbs/v1",
+        "https://identity.foundation/presentation-exchange/submission/v1"
+    ],
+    "type": [
+        "VerifiablePresentation",
+        "PresentationSubmission"
+    ],
+    "presentation_submission": {
+        "descriptor_map": [{
+            "id": "citizenship_input",
+            "path": "$.verifiableCredential.[0]"
+        }]
+    },
+    "verifiableCredential": [{
+        "@context": "https://www.w3.org/2018/credentials/v1",
+        "id": "https://eu.com/claims/DriversLicense",
+        "type": ["EUDriversLicense"],
+        "issuer": "did:foo:123",
+        "issuanceDate": "2010-01-01T19:73:24Z",
+        "credentialSubject": {
+            "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+            "license": {
+                "number": "34DGE352",
+                "dob": "07/13/80"
+            }
+       },
+       "proof": {
+           "type": "BbsBlsSignature2020",
+           "created": "2020-04-25",
+           "verificationMethod": "did:example:489398593#test",
+           "proofPurpose": "assertionMethod",
+           "requiredRevealStatements": [ 4, 5 ],
+           "signature": "F9uMuJzNBqj4j+HPTvWjUN/MNoe6KRH0818WkvDn2Sf7kg1P17YpNyzSB+CH57AWDFunU13tL8oTBDpBhODckelTxHIaEfG0rNmqmjK6DOs0/ObksTZh7W3OTbqfD2h4C/wqqMQHSWdXXnojwyFDEg=="
```

Comment:
should also have a nonce property.

Reply:
Note: I removed `nonce` from `options`.

`nonce` seems like a concern for the Holder and not the Verifier.

Reply:
@tplooker @kdenhartog any thoughts on if there should be a `nonce` option coming from the verifier or should nonce simply be generated by the holder?